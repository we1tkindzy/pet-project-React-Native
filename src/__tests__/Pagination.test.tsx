import {it} from '@jest/globals';
import renderer from 'react-test-renderer';
import Pagintaion from '../components/Pagination';

it('should renders Pagintaion component with a second active page', () => {
  const pagination = renderer.create(
    <Pagintaion pageNumber="2" setPageNumber={jest.fn()} />,
  );
  expect(pagination).toMatchSnapshot();
});
