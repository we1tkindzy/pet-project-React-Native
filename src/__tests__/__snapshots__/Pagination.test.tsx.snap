// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should renders Pagintaion component with a second active page 1`] = `
<View
  style={
    [
      {
        "borderColor": "#371005",
        "borderWidth": 1,
        "flexDirection": "row",
      },
      {
        "borderColor": "#371005",
      },
    ]
  }
>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      {
        "borderColor": "#371005",
        "borderWidth": 1,
        "opacity": 1,
        "paddingHorizontal": 15,
        "paddingVertical": 10,
      }
    }
  >
    <Text
      style={
        [
          {
            "color": "#B56735",
          },
          false,
        ]
      }
    >
      1
    </Text>
  </View>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      {
        "borderColor": "#371005",
        "borderWidth": 1,
        "opacity": 1,
        "paddingHorizontal": 15,
        "paddingVertical": 10,
      }
    }
  >
    <Text
      style={
        [
          {
            "color": "#B56735",
          },
          {
            "color": "#FFFFFF",
          },
        ]
      }
    >
      2
    </Text>
  </View>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      {
        "borderColor": "#371005",
        "borderWidth": 1,
        "opacity": 1,
        "paddingHorizontal": 15,
        "paddingVertical": 10,
      }
    }
  >
    <Text
      style={
        [
          {
            "color": "#B56735",
          },
          false,
        ]
      }
    >
      3
    </Text>
  </View>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      {
        "borderColor": "#371005",
        "borderWidth": 1,
        "opacity": 1,
        "paddingHorizontal": 15,
        "paddingVertical": 10,
      }
    }
  >
    <Text
      style={
        [
          {
            "color": "#B56735",
          },
          false,
        ]
      }
    >
      4
    </Text>
  </View>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      {
        "borderColor": "#371005",
        "borderWidth": 1,
        "opacity": 1,
        "paddingHorizontal": 15,
        "paddingVertical": 10,
      }
    }
  >
    <Text
      style={
        [
          {
            "color": "#B56735",
          },
          false,
        ]
      }
    >
      5
    </Text>
  </View>
</View>
`;
